### PR TITLE
Adding first stab at goals for 2017

### PR DESCRIPTION
**Pull request by kris-nova**
CC @kubernetes/sig-aws

### CODE REVIEW

**Comment by philips** [comment]
I like these goals :). @colhom has been trying to do the network thing.

File: sig-aws/2017_goals.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * We will consolidate networking reports and metrics around Kubernetes e2e tests and grow our metric list over time
+
+##### Statement
+
+A user can go to a single public location, and see a side by side comparison of networking avenues in Kubernetes on AWS.
+
+
+# Federation
+
+* We need to improve and harden our federation support in AWS
```
Comment (chrislovecnm) [inline sig-aws/2017_goals.md]: @justinsb ideas on what we can do here?

**Comment by chrislovecnm** [comment]
Bump

**Comment by kris-nova** [comment]
@chrislovecnm can you please make suggestions of what you are thinking? I am not quite understanding what/why we need munger, etc..

File: sig-aws/2017_goals.md (hunk 1 above)
Comment (chrislovecnm) [inline sig-aws/2017_goals.md]: @justinsb you mentioned a few more on our last sig call.  

@kris-nova how can I assist?  The notes are in the last meeting.  Should we review this next sig?

**Comment by chrislovecnm** [comment]
@kris-nova I was off, wrong repo, don't need formatting :)  Do we want to put in a design prop for this and have the community respond??

**Comment by kris-nova** [comment]
This is just static content that we kind of needed merged in a while ago.. If there are major concerns with the content can I suggest a follow up PR? We need to be able to reference our goals for the year as we drive features. I don't understand why we are pushing back on this.

**Comment by philips** [comment]
@kris-nova should this be merged?

**Comment by justinsb** [comment]
Pushing the button.  We can iterate if we need to.

**Pull request merged**